projects/ng-select2-component/src/lib/select2.component.html

```diff
@@ -74,7 +74,12 @@
                        spellcheck="false"
                        [attr.tabindex]="this.isOpen ? tabIndex : '-1'">
             </div>
-            <div class="select2-results">
+            <div class="select2-results"
+                 infiniteScroll
+                 [infiniteScrollDistance]="2"
+                 [infiniteScrollThrottle]="50"
```

Comment:
It would be better if `infiniteScrollDistance` and `infiniteScrollThrottle` were customable with default values

Reply:
On it!